src/aap_eda/api/serializers/auth.py

```diff
@@ -134,10 +136,10 @@ class RoleDetailSerializer(serializers.Serializer):
     created_at = serializers.SerializerMethodField()
     modified_at = serializers.SerializerMethodField()
 
-    def get_created_at(self, obj):
+    def get_created_at(self, obj) -> datetime.datetime:
         return obj.created
 
-    def get_modified_at(self, obj):
+    def get_modified_at(self, obj) -> datetime.datetime:
```

Comment:
Could you add docstrings for these methods that would knock out one of the [review comments](https://github.com/ansible/eda-server/pull/710#discussion_r1553425548) in feature/rbac PR, since you are already touching this line of code?

Reply:
This might sound pedantic, but I'd rather keep it out of the scope here since this patch doesn't change anything about it. Also, it's a bit hard for me to write that. I suspect whatever I write would be subject to changes and later revision. There are problem more problems with these endpoints that we haven't digested.